modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotPartitionsVerifyHandler.java

```diff
@@ -437,23 +434,23 @@ private PartitionHashRecord calculateDumpedPartitionHash(Dump dump, String grpNa
     /** {@inheritDoc} */
     @Override public void complete(String name,
         Collection<SnapshotHandlerResult<Map<PartitionKey, PartitionHashRecord>>> results) throws IgniteCheckedException {
-        Map<PartitionKey, List<PartitionHashRecord>> clusterHashes = new HashMap<>();
-        Map<ClusterNode, Exception> errs = new HashMap<>();
```

Comment:
Looks like we can use `bldr.exceptions()` everywhere for collection errors. Do we need separate `addException` method?

Reply:
I think not. Exceptions are processed and added differently. There are several places where we get exceptions as a collections, one by one. And we do not want create and wrap to a map manually. This is the target job of the builder.